radiomics/imageoperations.py

```diff
@@ -13,42 +13,64 @@
 logger = logging.getLogger(__name__)
 
 
-def getHistogram(binwidth, parameterValues):
-  """
+def getBinEdges(binwidth, parameterValues):
```

Comment:
> do the other options yield equally spaced bins?

I didn't test, but looking at the code, it seems that the binning strategy just calculates the width of the bin: https://github.com/numpy/numpy/blob/v1.12.0/numpy/lib/function_base.py#L695